srcpkgs/amule/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'amule'
+pkgname=amule
+version=11066
+revision=1
+wrksrc=amule-project-amule-9fd038f
+build_style=gnu-configure
+configure_args="--enable-cas --enable-wxcas
+ --enable-amule-daemon --enable-amulecmd
+ --enable-amule-gui --enable-alc  --enable-alcc
+ --enable-webserver --disable-debug --enable-optimize
+ --enable-geoip --disable-upnp --enable-fileview
+ --with-boost  --with-wx-config=/usr/bin/wx-config-gtk3"
+hostmakedepends="tar autoconf automake gettext gettext-devel"
+makedepends="boost zlib-devel wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
+depends="wxWidgets-gtk3 gd geoip crypto++"
+short_desc="Emule-like client for the eD2k and Kademlia networks"
+maintainer="hazayan <hazayan@hotmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/amule-project/amule"
+_commit="9fd038f8208f085dc7e0b52b73b2e18b6536437b"
```

Comment:
put that below `revision` so it's near the meta data you'd change on an update (grep for _githash in the repo)

Reply:
Fixed